Tasks/Korotenko/Task1/task1.py

```diff
@@ -0,0 +1,19 @@
+amount = int(input('Enter amount \n'))
+term = int(input('Enter time period(years) \n'))
+rate = float(input('Enter rate \n'))
+                        #currens = input('Enter currency \n')
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point